examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
<p>Overall needs a lot comments to be more "educational", then we could have another iteration</p><p> </p><ul><li><code>injection</code>&nbsp;that inject values at one or more given sparse positions (i.e point sources)</li><li><code>interpolation</code>&nbsp;that reads the value of a field at one or more given sparse positions (i.e point measurments)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>